frontend/survey-project/src/app/components/shared/ui/navbar/navbar.component.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["navbar.component.scss","navbar.component.css"],"names":[],"mappings":"AAAA;EACI,cAAA;ACCJ;;ADEA;EACI,iBAAA;EACA,gBAAA;EACA,wDAAA;EACA,gBAAA;EACA,eAAA;EACA,4BAAA;EACA,wCAAA;EACA,kBAAA;EACA,WAAA;ACCJ;ADCI;EACI,oBAAA;EACA,mBAAA;EACA,kBAAA;EACA,cAAA;EACA,mBAAA;EACA,2BAAA;ACCR;ADCQ;EACI,WAAA;EACA,YAAA;EACA,sBAAA;KAAA,mBAAA;ACCZ;ADGI;EACI,aAAA;EACA,mBAAA;EACA,mBAAA;EACA,qBAAA;EACA,UAAA;EACA,SAAA;ACDR;ADGQ;EACI,iBAAA;EACA,iBAAA;ACDZ;ADGY;EACI,qBAAA;EACA,WAAA;EACA,eAAA;EACA,gBAAA;ACDhB;;ADOA;EACI,eAAA;EACA,gBAAA;EACA,WAAA;EACA,qBAAA;EACA,+BAAA;EACA,yBAAA;EACA,kBAAA;ACJJ;;ADOA;EACI,aAAA;EACA,2BAAA;EACA,mBAAA;ACJJ","file":"navbar.component.css"}
```

Comment:
Not sure what is this used for, but I think it's not needed, we're using scss not css and the Angular builder will build our scss to css.